backend/internal/compile/daemon.go

```diff
@@ -297,5 +339,10 @@ func (app *CompiledApp) Request(ctx context.Context, method string, reqPath stri
 		return nil, fmt.Errorf("failed to make app request: %s", respBody.Error)
 	}
 
+	if !app.keepAliveRunning {
+		app.keepAliveRunning = true
+		go app.keepAlive()
+	}
```

Comment:
When would this trigger?

Reply:
This is when you start robin, and start using the app, so it sends a request to the Damon